app/views/shops/_edit_form.html.erb

```diff
@@ -0,0 +1,66 @@
+<%= form_for @shop, url: { action: :update }, html: { class: "col s8 offset-s2", id: "shop-edit-form" } do |f| %>
+<% if @shop.errors.any? %>
+<script>
+var msg = $("<ol><%= shop_form_errors(@shop.errors.full_messages).html_safe %></ol>");
+Materialize.toast(msg)
+</script>
+<% end %>
+<div class="row">
```

Comment:
## Nitpick- Code Quality
Embedding a script tag in html may not be the best option. Consider using rails built-in ajax request